### PR TITLE
Feature make suggestion

### PR DESCRIPTION
**Pull request by JerroldJV**

### CODE REVIEW

**Comment by kchason** [comment]
I see the form, but once it says "What's your accusation?" is there something else supposed to be there?

Also, the page becomes very wide. The "container" class has a default width of I believe 960px which causes problems. We can either remove it, or change it to container-fluid.

**Comment by kchason** [comment]
I pushed the container to container-fluid change

**Comment by JerroldJV** [comment]
@kchason I still haven't implemented accusations yet, but suggestions should be implemented.

File: clueless/templates/clueless/makeSuggestion.html [hunk 1]
```diff
@@ -1,8 +1,48 @@
...
+	 <br>
+    <h4>Room: {{ room.name }}</h4>
+	 <br>
+	  <label for="suggestion_suspect">Select suspect:</label>
+		<select class="form-control" name="suspect_id" id="suggestion_suspect">
+            {% for csi in characterSheetItems %}
+                <option value="{{ csi.card.card_id }}" aria-hidden="true">
+                    {% if csi.initiallyDealt %}
+                        {{ csi.card.name }} &nbsp;&nbsp;[dealt initially]
+                    {% elif csi.checked and csi.manuallyChecked %}
```
Comment (rsfriedman) [inline clueless/templates/clueless/makeSuggestion.html:17]: I'd like to change [dealt initially] to like [card is in hand] or something slightly more self-explanatory.

**Pull request merged**